src/de/jost_net/JVerein/gui/parts/BuchungsklasseSaldoList.java

```diff
@@ -475,14 +475,19 @@ else if (suBukSteuersatz.containsKey(buchungsart.getID()))
         suBukSteuer.put(steuersatz,
             suBukSteuer.get(steuersatz) + einnahmen + ausgaben + umbuchungen);
       }
-
-      zeile.add(new BuchungsklasseSaldoZeile(BuchungsklasseSaldoZeile.DETAIL,
-          buchungsart, einnahmen, ausgaben, umbuchungen));
+      if (Math.abs(einnahmen) >= LIMIT || Math.abs(ausgaben) >= LIMIT
+          || Math.abs(umbuchungen) >= LIMIT
+          || !Einstellungen.getEinstellung().getUnterdrueckungOhneBuchung())
```

Comment:
Kanst du das in ein Prädikat überführen? Das wäre einfacher zu lesen und zu verstehen.
So in der Art
```
if (!buchungUnterdruecken(einnahmen, ausgaben, umbuchung))
```

Oder wenn es etwas ohne Negation gibt, noch besser.

Reply:
Und den Einstellungscheck am Anfang durchführen, damit die anderen Checks gar nicht erst durchgeführt werden, falls die Unterdrückung aktiv ist.

Reply:
> Kanst du das in ein Prädikat überführen? Das wäre einfacher zu lesen und zu verstehen.

Grundsätzlich hast du recht. Nur deswegen extra eine Funktion einführen, wo dann der gleiche Check drin steht, halte ich für einmalige Verwendung nicht für nötig.